code/modules/antagonists/blob/overmind.dm

```diff
@@ -162,44 +164,64 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 		priority_announce("Confirmed outbreak of level 5 biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", ANNOUNCER_OUTBREAK5)
 		has_announced = TRUE
 
+/// Create a blob spore and link it to us
+/mob/camera/blob/proc/create_spore(turf/spore_turf, spore_type = /mob/living/basic/blob_minion/spore/minion)
+	var/mob/living/basic/blob_minion/spore/spore = new spore_type(spore_turf)
+	assume_direct_control(spore)
+	return spore
+
+/// Give our new minion the properties of a minion
+/mob/camera/blob/proc/assume_direct_control(mob/living/minion)
+	minion.AddComponent(/datum/component/blob_minion, src)
+
+/// Add something to our list of mobs and wait for it to die
+/mob/camera/blob/proc/register_new_minion(mob/living/minion)
+	blob_mobs |= minion
+	if (!istype(minion, /mob/living/basic/blob_minion/blobbernaut))
+		RegisterSignal(minion, COMSIG_LIVING_DEATH, PROC_REF(on_minion_death))
+
+/// When a spore (or zombie) dies then we do this
+/mob/camera/blob/proc/on_minion_death(mob/living/spore)
+	SIGNAL_HANDLER
+	blobstrain.on_sporedeath(spore)
+
 /mob/camera/blob/proc/victory()
 	sound_to_playing_players('sound/machines/alarm.ogg')
 	sleep(10 SECONDS)
-	for(var/i in GLOB.mob_living_list)
-		var/mob/living/L = i
-		var/turf/T = get_turf(L)
-		if(!T || !is_station_level(T.z))
+	for(var/mob/living/live_guy as anything in GLOB.mob_living_list)
+		var/turf/guy_turf = get_turf(live_guy)
+		if(!guy_turf || !is_station_level(guy_turf.z))
 			continue
 
-		if(L in GLOB.overminds || (L.pass_flags & PASSBLOB))
+		if(live_guy in GLOB.overminds || (live_guy.pass_flags & PASSBLOB))
 			continue
 
-		var/area/Ablob = get_area(T)
-
-		if(!(Ablob.area_flags & BLOBS_ALLOWED))
+		var/area/blob_area = get_area(guy_turf)
+		if(!(blob_area.area_flags & BLOBS_ALLOWED))
 			continue
 
-		if(!(ROLE_BLOB in L.faction))
-			playsound(L, 'sound/effects/splat.ogg', 50, TRUE)
-			if(L.stat != DEAD)
-				L.investigate_log("has died from blob takeover.", INVESTIGATE_DEATHS)
-			L.death()
-			new/mob/living/simple_animal/hostile/blob/blobspore(T)
+		if(!(ROLE_BLOB in live_guy.faction))
+			playsound(live_guy, 'sound/effects/splat.ogg', 50, TRUE)
+			if(live_guy.stat != DEAD)
+				live_guy.investigate_log("has died from blob takeover.", INVESTIGATE_DEATHS)
+			live_guy.death()
+			create_spore(guy_turf)
 		else
-			L.fully_heal()
+			live_guy.fully_heal()
 
-		for(var/area/A in GLOB.areas)
-			if(!(A.type in GLOB.the_station_areas))
+		for(var/area/check_area in GLOB.areas)
+			if(!(check_area.type in GLOB.the_station_areas))
```

Comment:
while we're here can you change this to use the dedicated proc, `is_type_in_list()`?